vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -70,6 +71,8 @@ def pre_build(self):
             dash.register_page(
                 module=page.id,
                 name=page.title,
+                description=page.description,
+                image=self._infer_image("app") or self._infer_image("logo"),
```

Comment:
Right! I forgot why I did that, and it looks prettier here.  I'll change it back :-)

Reply:
To be honest I probably would have written exactly as you did it here also, because everything is is done inline so it looks nicer. I didn't even think of putting it outside the loop until you did so, but I think it is a good idea because recursive through directories is a very slightly expensive operation, and there's no need for us to repeat it for every page.